rust/toolchain.bzl

```diff
@@ -268,6 +268,15 @@ def _rust_toolchain_impl(ctx):
         linking_context = linking_context,
     )
 
+    # Calculate the rustc sysroot path by using a file from the rust-std bundle
+    rust_std_files_list = rust_std.files.to_list()
+    sysroot_anchor = rust_std_files_list[0]
+    sysroot_path_parts = sysroot_anchor.path.split("/")
+    sysroot_path = ""
+    if len(sysroot_path_parts) > 0:
```

Comment:
I think the current behavior is more cleanly expressed using `rpartition`?

Reply:
Oh, thanks! Updated